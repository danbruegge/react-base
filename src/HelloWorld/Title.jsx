// @flow
import styled from 'styled-components';

export default styled.h1`
    background-color: SeaGreen;
    color: #fff;
    font-size: 40px;
    margin-bottom: 5px;
    padding-left: 5px;
`;
