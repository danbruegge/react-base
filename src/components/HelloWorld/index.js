import React from 'react';

import { Title, Message } from 'app/components/core';

const HelloWorld = () => (
    <div>
        <Title>Hello World</Title>
        <Message>This is a hello world message.</Message>
    </div>
);

export default HelloWorld;
