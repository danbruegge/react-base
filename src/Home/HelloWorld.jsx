import React from 'react';

import Title from './Title';
import Message from './Message';

export default () => (
    <div>
        <Title>Hello World</Title>
        <Message>This is a hello world message.</Message>
    </div>
);
