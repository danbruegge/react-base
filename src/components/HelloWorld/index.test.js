import React from 'react';
import renderer from 'react-test-renderer';

import HelloWorld from './index';

describe('HelloWorld', () => {
    test('Should render component', () => {
        const component = <HelloWorld />;
        const tree = renderer.create(component).toJSON();

        expect(tree).toMatchSnapshot();
    });
});
