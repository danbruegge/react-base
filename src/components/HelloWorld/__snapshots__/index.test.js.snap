// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld Should render component 1`] = `
<div>
  <h1
    className="sc-bdVaJa dgExSG"
  >
    Hello World
  </h1>
  <p
    className="sc-bwzfXH KXrTm"
  >
    This is a hello world message.
  </p>
</div>
`;
